metalibs/otp/otp.cxx

```diff
@@ -7,3 +7,11 @@
 // something to compile when making libtoontown.so in NO_DEFER mode,
 // which generates an empty library that itself links with all the
 // other shared libraries that make up libtoontown.
+
+// This is the one file in this directory which is actually compiled.  It
+// exists just so we can have some symbols and make the compiler happy.
+
+#if defined(_WIN32) && !defined(CPPPARSER) && !defined(LINK_ALL_STATIC)
+__declspec(dllexport)
+#endif
+int otp
```

Comment:
You forgot a semicolon at the end of this line lol.  Also, the end of the file needs an empty newline.